src/D2L.Services.Core.Extensions/ParallelizationExtensions.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;

```

Comment:
Really nice use of advanced async-await, generics and functional programming in this class!